### DISCUSSION TITLE
Question about Creating a basic Astroid Template

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#147] [General]
Hi, i create a basic template like the docs here: https://astroidframe.work/support/documentations/getting-started/creating-a-basic-astroid-template
That works fine. I use the Template One to copy the template.
But, when you update Template One, did my own template automatic update too?
It looks like, the present profile has some probs (it looks like in One and in Zero), and i hope my own basic template was fixing too.
With best regards,
deltapapa

**Comment by sonvnn** [comment]
Hi @deltapapa01 

I updated One template. You can download One Template at https://github.com/templaza/astroid-framework/archive/refs/heads/one-template.zip

Best Regards,
Sonny

**Comment by deltapapa01** [comment]
Hi, okay i check that out. My own basic template no matter whether with zero or one, are not updated or the present function still does not work there. With the original templates this works wonderfully now.
